### ISSUE TITLE
graph and data is slightly mismatched. 

### ISSUE THREAD

**Issue by kotran88** [almende/vis#4137]
as you can see below link,( it's basic example of graph )

x is 2014-06-11  and y is 10 
but square shown on pic should be moved to left to say that 11(2014-06-11) is 10. 

and 2014-06-13 is 30. but it's also slightly leaning to the right.

how can I make graph more matched exactly as I wished? 

[https://imgur.com/a/IuMqRCG](url)

**Comment by DavidSpackman** [comment]
Is it offset by a timezone difference?

**Comment by kotran88** [comment]
> Is it offset by a timezone difference?

I don't think it 's about offset by timezone...because linked image is the one that I get from official website's example...isn't it wired to read graph? it's not matched x axis and y axis exactly. leaning to right a bit.

**Comment by DavidSpackman** [comment]
Can you test with full ISO8601 Datetime with time zone ?
i.e. I'm in GMT+8, using the data below displays the data on the graph correctly for me.

  var items = [
    {x: '2014-06-11T00:00:00+08:00', y: 10},
    {x: '2014-06-12T00:00:00+08:00', y: 25},
    {x: '2014-06-13T00:00:00+08:00', y: 30},
    {x: '2014-06-14T00:00:00+08:00', y: 10},
    {x: '2014-06-15T00:00:00+08:00', y: 15},
    {x: '2014-06-16T00:00:00+08:00', y: 30}
  ];
